ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -265,6 +265,28 @@ StatusDialog {
             store.setSendType(popup.preSelectedSendType)
         }
 
+        if(d.isBridgeTx) {
```

Comment:
there is tonne of simplification needed as this is not need for needing anymore. I guess it's not your aim to achieve that here? also on the nim side.

Reply:
We also need to remove all the logic on him and add UI state to UI :(